src/console_reporter.cc

```diff
@@ -100,7 +115,22 @@ void ConsoleReporter::PrintRunData(const Run& result) {
   ColorPrintf(COLOR_GREEN, "%-*s ",
```

Comment:
I have changed the color to blue, but I do not have any particular preference. So if you have any preference I can change it to any other color. With respect to the output, I actually only use the CPU, but as on the reports the real time is given I supposed that somebody might be interested on it, maybe for some test where CPU value is not reliable, I don´t know... I think it does not heart to have it there but you know better than me the users of the tool, so if you think is not useful, I could just remove it.